### DISCUSSION TITLE
Should we have "font-smoothing" set to "antialiased" or should we leave it unset?

### DISCUSSION THREAD

**Discussion by cdignam-segment** [twilio-labs/paste#3513] [Q&A] [answered]
In the Segment App, we have this ancient setting to enable font-smoothing, which causes the fonts to look a little different from designs. Should this be removed to be more in line with Paste designs?

https://github.com/segmentio/app/blob/b086b6ed4aeb75a79b6efa548dc22590012cdefe/packages/app/client/index.css#L24-L27

```css
  /* Fix the thick text rendering on OSX */
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-smoothing: antialiased;
```

**Comment by SiTaggart** [comment, marked as answer]
Hi @cdignam-segment 

Ideally, yeah, that would be something that gets removed from the app at some point. I don't think it needs to removed any time soon, the app is using evergreen and evergreen themed Paste which uses the same font as evergreen. So that font smoothing should be acting the same on both and be totally fine.

Once we switch to the Twilio theme and we change the font to Twilio sans, then we might need to remove it from the app, but I don't think we're there yet.

I wonder if you are seeing Figma designs using the Twilio theme?
